### PR TITLE
more spelling fixes

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

File: src/core/qgsmaplayer.cpp [hunk 1]
```diff
@@ -385,7 +385,7 @@ bool QgsMapLayer::readLayerXml( const QDomElement& layerElement )
   layerError = !readXml( layerElement );
 
   // overwrite CRS with what we read from project file before the raster/vector
-  // file readnig functions changed it. They will if projections is specfied in the file.
+  // file readnig functions changed it. They will if projections is specified in the file.
```
Comment (DelazJ) [inline src/core/qgsmaplayer.cpp]: reading

File: src/core/qgsmaprendererjob.h [hunk 2]
```diff
@@ -142,7 +142,7 @@ class CORE_EXPORT QgsMapRendererJob : public QObject
     //! @note The way how geometries are cached is really suboptimal - this method may be removed in future releases
     void setRequestedGeometryCacheForLayers( const QStringList& layerIds ) { mRequestedGeomCacheForLayers = layerIds; }
 
-    //! Find out how log it took to finish the job (in miliseconds)
+    //! Find out how log it took to finish the job (in milliseconds)
```
Comment (DelazJ) [inline src/core/qgsmaprendererjob.h]: long

File: src/core/qgssnappingconfig.h [hunk 3]
```diff
@@ -81,7 +81,7 @@ class CORE_EXPORT QgsSnappingConfig
         //! return if settings are valid
         bool valid() const;
 
-        //! return if snaping is enbaled
+        //! return if snaping is enabled
```
Comment (DelazJ) [inline src/core/qgssnappingconfig.h]: snapping

File: src/gui/symbology-ng/qgssymbolselectordialog.h [hunk 4]
```diff
@@ -215,7 +215,7 @@ class GUI_EXPORT QgsSymbolSelectorWidget: public QgsPanelWidget, private Ui::Qgs
     void updateLayerPreview();
 
     /**
-     * Update the preivew of the whole symbol in the iterface.
+     * Update the preivew of the whole symbol in the interface.
```
Comment (DelazJ) [inline src/gui/symbology-ng/qgssymbolselectordialog.h]: preview

File: src/plugins/evis/eventbrowser/evisgenericeventbrowsergui.cpp [hunk 5]
```diff
@@ -116,7 +116,7 @@ eVisGenericEventBrowserGui::~eVisGenericEventBrowserGui()
   QSettings settings;
   settings.setValue( QStringLiteral( "/eVis/browser-geometry" ), saveGeometry() );
 
-  //Clean up, disconnect the highlighting routine and refesh the canvase to clear highlighting symbol
+  //Clean up, disconnect the highlighting routine and refresh the canvase to clear highlighting symbol
```
Comment (DelazJ) [inline src/plugins/evis/eventbrowser/evisgenericeventbrowsergui.cpp]: canvas

File: src/plugins/plugin_template/plugingui.cpp [hunk 6]
```diff
@@ -66,7 +66,7 @@
                  .arg( tr( "This is a Qt designer 'ui' file. It defines the look of the default plugin dialog without implementing any application logic. You can modify this form to suite your needs or completely remove it if your plugin does not need to display a user form (e.g. for custom MapTools)." ) )
                  .arg( tr( "This is the concrete class where application logic for the above mentioned dialog should go. The world is your oyster here really...." ) )
                  .arg( tr( "This is the Qt4 resources file for your plugin. The Makefile generated for your plugin is all set up to compile the resource file so all you need to do is add your additional icons etc using the simple xml file format. Note the namespace used for all your resources e.g. (':/Homann/'). It is important to use this prefix for all your resources. We suggest you include any other images and run time data in this resurce file too." ) )
```
Comment (DelazJ) [inline src/plugins/plugin_template/plugingui.cpp]: **resource** file too (at the end)

**Comment by DelazJ** [comment]
@3nids Thanks to take the lead on spelling checks.
For my information, when you fix a typo, you also fix all occurences of this spelling in the repository, don't you? I mean, if any of the items reported in https://github.com/qgis/qgis3_UIX_discussion/issues/19 appears in your commit then it can be considered as fixed in the repo?

**Comment by 3nids** [comment]
yes, it should be!

**Comment by 3nids** [comment]
manually merged in bb9d536

**Pull request closed without merging**